data/tlg4170/tlg001c/__cts__.xml

```diff
@@ -0,0 +1,7 @@
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg4170" xml:lang="grc" urn="urn:cts:greekLit:tlg4170.tlg001c.1st1K-grc1">
```

Comment:
The `urn` attribute here is incorrect. This should be the work-level URN instead of the edition-level one. So delete `.1st1K-grc1` from the end of it.